zrml/prediction-markets/src/tests.rs

```diff
@@ -720,23 +721,24 @@ fn it_resolves_a_disputed_market() {
         // ---------------------------
         // - OracleBond: 50 * CENT
         // - Dave's reserve: DisputeBond::get() + DisputeFactor::get()
-        // Total: 50 * CENT + DisputeBond::get() + DisputeFactor::get()
-        // Per each: 25 * CENT + (DisputeBond::get() + DisputeFactor::get()) / 2
+        // Total: OracleBond::get() + DisputeBond::get() + DisputeFactor::get()
+        // Per each: ( OracleBond::get() + DisputeBond::get() + DisputeFactor::get()) / 2
```

Comment:
I agree, because Eve and Charlie won. So they are two and the total reward is divided by two.